src/Form.svelte

```diff
@@ -104,8 +104,10 @@
               <span>
                 <a class="link" href="{field.value}" target="_blank">{field.value}</a>
               </span>
+            {:else if field.name === "owner"}
+              <Address address={field.value} resolve {config} />
```

Comment:
The 'Form' component shouldn't change its behavior based on field names, this breaks separation of concerns -- rather the parent component should configure the form to display the "owner" field a certain way.

We can add a `resolve` attribute to fields, for example.